master/networking/dual-stack.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Enable dual stack
+canonical_url: 'https://docs.projectcalico.org/master/networking/dual-stack'
+---
+
+### Big picture
+
+Arrange for each Kubernetes pod to get both IPv4 and IPv6 addresses, so that it can communicate over
+both IPv4 and IPv6.
+
+### Value
+
+Communication over IPv6 is increasingly desirable, and the natural approach for cluster pods is to
+be IPv6-native themselves, as well as still supporting IPv4.  Native support for both IPv4 and IPv6
+is known as "dual stack", and Kubernetes has alpha-level support for this in versions 1.16 and 1.17.
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- [**CNI plugin configuration**]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration#ipam) with `assign_ipv6: true`
+
+### Before you begin...
+
+You must first set up a cluster following the Kubernetes
```

Comment:
1. Set up a cluster following the Kubernetes
[prerequisites](https://kubernetes.io/docs/concepts/services-networking/dual-stack/#prerequisites)
and [enablement
steps](https://kubernetes.io/docs/concepts/services-networking/dual-stack/#enable-ipv4-ipv6-dual-stack)
for dual stack, but ignore mention of the Kubenet network plugin, because here we will use {{site.prodname}} instead.
2. Follow our [installation
guide]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/calico) to identify
and download the correct {{site.prodname}} manifest for the cluster, and
for your preferred datastore type, but do not apply that manifest yet.

Reply:
(I've just retained "except" instead of "but", as I think it's a slightly stronger warning flag word.)